src/error_handler.coffee

```diff
@@ -0,0 +1,48 @@
+define [
+  'jquery'
+  'primedia_events'
+  'src/formatter'
+  'jquery.cookie'
+], (
+  $
+  events
+  Formatter
+) ->
+  class ErrorHandler
+
+    constructor: (@error, @$box, @eventName) ->
+
+    generateErrors: () ->
+      @clearErrors()
+      messages = ''
+      if error?
+        $form = @$box.parent().find 'form'
+        $.each @error, (key, value) =>
+          $form.find("##{key}").parent('p').addClass 'error'
+          formattedError = @_formatError key, value
+          messages += "<li>#{formattedError}</li>"
+          $form.find('.error input:first').focus()
+      else
+        messages += "An error has occurred."
+      @$box.append "<ul>#{messages}</ul>"
+      events.trigger('event/' + eventName, error)
+
+
+    _formatError: (key, value) ->
```

Comment:
What are your thoughts about removing the underscore? It makes more sense for us to treat this as a public function on an `ErrorHandler` object than it did when this function lived on the `Login` object.